HospitalGame_Business/src/business/BusinessFacade.java

```diff
@@ -5,128 +5,206 @@
  */
 package business;
 
+import business.Item.BloodBag;
+import business.Item.ItemFacade;
+import business.Item.PowerUpItem;
+import business.NPC.Computer;
+import business.NPC.Doctor;
+import business.NPC.NPCFacade;
+import business.NPC.Porter;
 import business.common.IData;
 import business.common.IItemFacade;
 import business.common.INPCFacade;
+import common.BloodType;
+import common.Directions;
+import common.GameConstants;
 import common.IBusiness;
 import common.IHighScore;
+import common.IItem;
 import common.INPC;
 import common.IPersistence;
 import common.IPlayer;
+import common.ItemName;
+import common.NPCID;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Random;
 
 /**
  *
  * @author andreasmolgaard-andersen
  */
 public class BusinessFacade implements IBusiness {
 
+    private ArrayList<IItem> items = new ArrayList<>();
+
     /**
      * to gain access to item facade
      */
     private IItemFacade itemFacade;
-    
+
     /**
      * to gain access to npc facade
      */
     private INPCFacade npcFacade;
-    
+
     /**
      * gives access to data facade
      */
     private IData dataFacade;
-    
+
     /**
      * is the player for business facade to manage
      */
     private Player player;
-    
+
     /**
      * is the map of the game to hold
      */
     private Map map;
-    
+
     /**
      * high score to save score of the game
      */
     private BusinessHighScore highScore;
-    
+
     /**
      * to gain access to persistence
      */
     private IPersistence persistence;
-    
+
     /**
      * a
      */
     private boolean isGameOver = false;
-    
+
+    public BusinessFacade() {
+        map = new Map();
+        itemFacade = new ItemFacade();
+        npcFacade = new NPCFacade();
+        map.injectItemFacade(itemFacade);
+        map.InjectNPCFacade(npcFacade);
+        
+    }
+
+    public final ArrayList<INPC> makeUpNpcs() {
```

Comment:
What is this used for?
And if it is used in the facade only it should be _private_